soh/soh/SaveManager.cpp

```diff
@@ -1039,14 +1039,28 @@ void SaveManager::LoadFile(int fileNum) {
     switch (saveBlock["version"].get<int>()) {
         case 1:
             for (auto& block : saveBlock["sections"].items()) {
-                int sectionVersion = block.value()["version"];
                 std::string sectionName = block.key();
                 if (!sectionLoadHandlers.contains(sectionName)) {
                     // Unloadable sections aren't necessarily errors, they are probably mods that were unloaded
                     // TODO report in a more noticeable manner
                     SPDLOG_WARN("Save " + GetFileName(fileNum).string() + " contains unloadable section " + sectionName);
                     continue;
                 }
+                if (!block.value().contains("version")) {
+                    // We're trying to load a section without a version, this means this
+                    // section of the save file is invalid
+                    // Report the error so that the user can rectify the error.
+                    // TODO report in a more noticeable manner
+                    SPDLOG_ERROR("Save " + GetFileName(fileNum).string() + " contains section " + sectionName +
+                                 " with NO version");
+                    if (sectionName == "sohStats") {
+                        SPDLOG_WARN("Loading save without stats section...");
+                        continue;
+                    }
+                    assert(false);
+                    continue;
```

Comment:
Is this just to prevent sohStats from tripping the debug assert? Any reason why we would want that?

Reply:
It was an idea for a workaround because people were crashing on boot because of missing sohstats in the save. If we have plans that should prevent saves getting into that broken state I can close this